### PR TITLE
Add harbor package configuration table in README

### PR DESCRIPTION
**Pull request by MinerYang**
## What this PR does / why we need it
Add harbor package configuration table in README

## Describe testing done for PR
- mdlint check
- mispell check

### CODE REVIEW

**Review by heww — APPROVED** [review]

File: addons/packages/harbor/2.2.3/README.md [hunk 1]
```diff
@@ -197,3 +184,243 @@ Users can now connect to the Harbor UI by navigating to `https://harbor.yourdoma
    ```shell
    docker pull harbor.yourdomain.com/library/nginx:1.7.9
    ```
+
+## Configuration
+
+The follwing lightwight pass-through values can be set to customize the Harbor installation.
```
Comment (kcoriordan) [inline addons/packages/harbor/2.2.3/README.md:190]: typo: lightweight

File: addons/packages/harbor/2.3.3/README.md [hunk 2]
```diff
@@ -183,3 +168,255 @@ Users can now connect to the Harbor UI by navigating to `https://harbor.yourdoma
    ```shell
    docker pull harbor.yourdomain.com/library/nginx:1.7.9
    ```
+
+## Configuration
+
+The follwing lightwight pass-through values can be set to customize the Harbor installation.
```
Comment (kcoriordan) [inline addons/packages/harbor/2.3.3/README.md:174]: typo: lightweight

File: addons/packages/harbor/2.2.3/README.md [hunk 3]
```diff
@@ -92,17 +82,15 @@ The Harbor package requires use of Contour for ingress, cert-manager for certifi
...
+   Or to generate them automatically, run
 
    ```shell
    bash generate-passwords.sh harbor-values.yaml
    ```
 
    This step is needed only once.
 
-1. Specify other Harbor configuration (e.g. admin password, hostname, persistence setting, etc.) in `harbor-values.yaml`.
+1. Specify other Harbor configuration (e.g. admin password, hostname, persistence setting, etc.) in `harbor-values.yaml`. To customize Harbor insatllation, you could refer to the pass-through value setting in `Configuration` section.
```
Comment (kcoriordan) [inline addons/packages/harbor/2.2.3/README.md:93]: typo: installation

File: addons/packages/harbor/2.3.3/README.md [hunk 4]
```diff
@@ -183,3 +168,255 @@ Users can now connect to the Harbor UI by navigating to `https://harbor.yourdoma
...
+|` persistence.imageChartStorage.disableredirect `| Specify whether to disable `redirect` for images and chart storage, for backends which not supported it (such as using minio for `s3` storage type), please disable it. To disable redirects, simply set `disableredirect` to `true` instead. Refer to <https://github.com/docker/distribution/blob/master/docs/configuration.md#redirect> for the detail. | false | boolean |
+|` persistence.imageChartStorage.caBundleSecretName `| Specify the "caBundleSecretName" if the storage service uses a self-signed certificate. The secret must contain keys named "ca.crt" which will be injected into the trust store of registry's and chartmuseum's containers. |  | string |
+
+#### *FileSystem*
+
+|Values | Description | Default | Type |
+|-------|-------------|---------|------|
+|` persistence.imageChartStorage.filesystem.rootdirectory `| The rootdirectory in filesystem. | /storage | string |
```
Comment (kcoriordan) [inline addons/packages/harbor/2.3.3/README.md:291]: typo: root directory

Comment (MinerYang) [reply addons/packages/harbor/2.3.3/README.md:291]: THX

**Review by heww — APPROVED** [review]

**Pull request merged**